docs/docs/cmd/planner/roster/roster-add.md

```diff
@@ -71,5 +71,6 @@ m365 planner roster add
     assignedSensitivityLabel | null
     ```
 
-## Additional information
+## More information
```

Comment:
Hi @milanholemans, in my understanding, 'More information' is typically used for links to external sources.
What you want here is the term 'Remarks'

Reply:
I don't really agree with this one. My perception of `Remarks` is information you should know to work with the command. Sort of `Remarks when using this command`. Here we can explain options a bit more, talk about permissions, ...

`More information` is more like additional information about the stuff you are doing with the command in M365 for example. In this case, working with Rosters. Since Rosters are a new thing, it might not be obvious to everyone what a Roster exactly is. If you know what a Roster is, you can perfectly skip this section. Unlike the Remarks section which always has useful info about the command.

At least that's how I see it 😊

I noticed there are some more commands which are just explaining stuff about M365 (without links). Example: https://pnp.github.io/cli-microsoft365/cmd/purview/retentionevent/retentionevent-list/#more-information

Reply:
I agree that the headers' naming conventions are inconsistent. We have "remarks," "additional information," and "more information," which is causing clutter. This PR provides an excellent to fix this.

Regarding terminology, I support @milanholemans opinion. "Remarks" should primarily cover special conditions related to the command. As for the section containing external links, I suggest renaming it to "External Links" and simply listing the links without extensive additional information.

For scenarios like `m365 purview retentionevent list` it's "More information" section. I think we should avoid including such details in our command documentation. Instead, it would be better to provide an external link to the corresponding MS Docs page.

Reply:
Hi @milanholemans, @Jwaegebaert,

Some discussion on this would be a good thing. 
I agree with you that remarks typically refers to using the command, so I'll withdraw the comment on that.

The question what we should do with these explanatory segments flies for the purview commands as well as this one. Maybe, like @Jwaegebaert says, we should just remove them. 

It would be an interesting broader discussion to see if we do want to include some more information like PnP PowerShell also does. But that's outside of the scope of this PR.

@milanholemans, would you agree to completely remove the section and just keep the link? I can remove the sections on purview event-based retention myself.

Reply:
By the way: I suggest that we do not rename the header to 'External links'. let's discuss that in a separate issue

Reply:
Yeah, good point. This discussion can get more extensive than the scope of this PR 😄

Reply:
Can we agree to just merge this as is, and discuss internally how we are going to deal with it? Otherwise, I might do things that the rest don't agree with and we can adjust it in an additional PR.

Reply:
That's okay with me as well. In that case I'll approve this one 👍